### ISSUE TITLE
Warn when content in extends child

### ISSUE THREAD

**Issue by softdevca** [rinja-rs/rinja#164]
Top-level content [is ignored](https://github.com/rinja-rs/rinja/blob/3747ae2b3436a5f0bb2d7e946ad9d831c139ef3b/book/src/template_syntax.md?plain=1#L378) in templates that extend another.  Rinja should warn when this is encountered instead of silently ignoring it to reduce confusion.

```html
This is ignored
{% extends "base.html" %}
This is also ignored
{% block content %}
    <p>Hello, world</p>
{% endblock %}
```

**Comment by Kijewski** [comment]
Yes, I think it would be better if we make this a syntax error! But I guess is would be a breaking change, because it's a documented feature that any data outside of `{% block %}` is ignored.
